src/style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+.label.searchbar {
+  width: 100%;
+  height: 70px;
+}
+
+.searchbar--big {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  margin-top: 20px;
+}
+
+.icon--big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  z-index: 1;
+
+}
+
+.input--big {
+  position: absolute;
+  text-indent: 62px;
+  height: 70px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 400;
+  color: #3d4e61;
+  border: hidden;
+}
+
+.searchbar--small {
+  position: relative;
+  margin-top: 20px;
+  width: 100%;
+  height: 42px;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.icon--small {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+  z-index: 1;
+}
+
+.input--small {
+  position: absolute;
+  text-indent: 33px;
+  height: 42px;
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 400;
+  color: #3d4e61;
+  border: none;
+}

```

Comment:
create `.icon` class and move common styles there

Reply:
please check all places (you can write common styles for `.searchbar` and it's states (:hover, :focus)as well